README.md

```diff
@@ -133,11 +133,13 @@ egjs has the dependencies for the following libraries:
 > Except jQuery, library dependencies may vary depending on modules. For more information, see API documentation.
 
 When add dependency library have to use `yarn`. 
-> yarn add: adds a package to use in your current package.
-> yarn init: initializes the development of a package.
-> yarn install: installs all the dependencies defined in a package.json file.
-> yarn publish: publishes a package to a package manager.
-> yarn remove: removes an unused package from your current package.
+```
+yarn add: adds a package to use in your current package.
+yarn init: initializes the development of a package.
+yarn install: installs all the dependencies defined in a package.json file.
+yarn publish: publishes a package to a package manager.
+yarn remove: removes an unused package from your current package.
+```
```

Comment:
@netil 
OK. I think yarn`s command unnecessarily is description too. But IMO somewhere add description that if you want to add dependency library then you have to use `yarn`.

Beacause. npm not support yarn env.
npm add AAA 
 - yarn : AAA uninstall
 - npm : AAA install
yarn add AAA
 - yarn : AAA install
 - npm : AAA install